Source/WebGPU/WebGPU/PipelineLayout.mm

```diff
@@ -78,6 +78,12 @@
     return false;
 }
 
+void PipelineLayout::makeInvalid()
+{
+    m_isValid = false;
+    m_bindGroupLayouts.clear();
```

Comment:
Why?

Reply:
Calling `pipelineLayout.bindGroupLayout(K)` on an invalid layout should return a value. Alternatively, we can check the flag in `PipelineLayout::bindGroupLayout(` - which is preferable?